bluebrain/repo-bluebrain/packages/py-brayns/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyBrayns(PythonPackage):
+    """Python client to the Brayns renderer"""
+
+    homepage = "https://github.com/BlueBrain/Brayns"
+    git = "https://github.com/BlueBrain/Brayns.git"
+
+    submodules = False
+
+    build_directory = 'python'
+
+    version('develop', branch='develop')
+    version('3.0.0', tag='3.0.0')
```

Comment:
Do you have PyPI source tarballs that could be used instead? Would be a little bit more robust.

Reply:
Added!